apps/full-site-editing/full-site-editing-plugin/global-styles/src/dom-updater.js

```diff
@@ -14,23 +15,32 @@ export default ( options, getOptionValue ) => {
 	domReady( () => {
 		const current = {};
 		const cssVariables = {};
+		const styleElement = document.createElement( 'style' );
+		document.body.appendChild( styleElement );
+		const styleSheet = styleElement.sheet;
+
 		options.forEach( option => {
 			current[ option ] = null;
 			cssVariables[ option ] = `--${ option.replace( '_', '-' ) }`;
 		} );
 
 		subscribe( () => {
+			let styleProps = '';
 			Object.keys( current ).forEach( key => {
 				const value = getOptionValue( key );
-				if ( current[ key ] !== value ) {
+				if ( ! isEmpty( value ) && current[ key ] !== value ) {
 					current[ key ] = value;
-					// We want to scope this to the root node of the editor.
-					const node = document.getElementsByClassName( 'editor-styles-wrapper' )[ 0 ];
-					if ( node ) {
-						node.style.setProperty( cssVariables[ key ], value );
-					}
+					styleProps += `${ cssVariables[ key ] }:${ value };`;
 				}
 			} );
+			// We want to scope this to the root node of the editor.
+			// We need this to be a stylesheet rather than inline styles so the styles apply to all editor instances incl. previews.
```

Comment:
Thanks for adding this comment. It'd be lovely if we could restrain comments' line length to 80 characters :)

Reply:
We can also remove the original comment as it no longer makes sense, though.

Reply:
> It'd be lovely if we could restrain comments' line length to 80 characters :)
Sure, done!

> We can also remove the original comment as it no longer makes sense, though.

We're still scoping to the root node of the editor though, no? As in it's the same class, it's just applying to all editor instances in the document. But can also remove that one if you prefer?

Reply:
Removed it :)